test/run-tests

```diff
@@ -110,13 +110,12 @@ def handle_transient_httperrors(error):
         except Exception:
             github_msg = ""
         else:
-            print("W: GitHub: " + github_msg)
+            logging.error("GitHub: %s", github_msg)
 
         if "abuse detection" in github_msg:
             timeout = transient_error_wait_time.next()
-            print(
-                ">>> We have triggered abuse detection, waiting {} s...".format(timeout)
-            )
+            msg = "We have triggered abuse detection, waiting {} s...".format(timeout)
+            logging.warning("%s", msg)
```

Comment:
`logging.warning("We have triggered abuse detection, waiting %d s...", timeout)`